tests/distributed/test_distributed_sampling.py

```diff
@@ -270,6 +275,21 @@ def create_random_hetero():
     g.nodes['n1'].data['feat'] = F.ones((g.number_of_nodes('n1'), 10), F.float32, F.cpu())
     return g
 
+def create_random_hetero_dense():
```

Comment:
how is this one different from above?

Reply:
This will create a more dense graph to ensure each relation type has more than 3 edges.

Reply:
can you make density as an argument and reuse the code?

Reply:
Done